frontend/storage/configureStore.ts

```diff
@@ -25,7 +25,12 @@ function createMockGamesVisibility(): Map<number, boolean> {
 function createMockGames(): Map<number, Game> {
 	let mockData: Map<number, Game> = new Map();
 	for (let i = 0; i < 20; i++) {
-		const mockGame = new Game({ id: i, name: `Super Metroid: ${i}` });
+		const mockGame = new Game({
+			id: i,
+			name: `Super Metroid: ${i}`,
+			series: 'Metroid',
+			imgUrl: 'https://steamuserimages-a.akamaihd.net/ugc/90469956945157107/05A816D9D18F48CEE9FE3E0AA8706B0BF81B669E/',
```

Comment:
Download the image and add it to the repository. [Here](https://github.com/EvidentlyCube/metroidvania-compendium-design/blob/master/src/components/common/AbilityIcon.jsx#L3) is how to get images into React.